packages/aws-cdk-lib/aws-route53/README.md

```diff
@@ -192,11 +192,39 @@ const parentZone = new route53.PublicHostedZone(this, 'HostedZone', {
   zoneName: 'someexample.com',
 });
 const crossAccountRole = new iam.Role(this, 'CrossAccountRole', {
-  // The role name must be predictable
-  roleName: 'MyDelegationRole',
-  // The other account
-  assumedBy: new iam.AccountPrincipal('12345678901'),
-});
+      // The role name must be predictable
```

Comment:
the formatting here is inconsistent with the rest of the doc in terms of tabs

Reply:
Not sure I follow, I see other comments in the doc follow the tab alignment of the line of code below it that it speaks to. Can you clarify?

Reply:
<img width="942" alt="Screenshot 2024-01-09 at 3 26 23 PM" src="https://github.com/aws/aws-cdk/assets/36202692/8f0276dd-7193-4af8-9830-1cb4e8c5e74b">


here is your rich diff. the tab alignment is off